pyquil/latex/ipython.py

```diff
@@ -0,0 +1,42 @@
+import os
+import subprocess
+import shutil
+import tempfile
+from IPython.display import Image
+from pyquil import Program
+from .latex_generation import to_latex
+
+
+def display(circuit: Program, settings=None, **image_options) -> Image:
+    """
+    Renders a PyQuil circuit as an IPython image object.
+
+    :param Program circuit: The circuit to be drawn, represented as a pyquil program.
+    :param DiagramSettings settings: An optional object of settings controlling diagram rendering and layout.
+    :return: PNG image render of the circuit.
+    :rtype: Image
+    """
+    pdflatex_path = shutil.which('pdflatex')
```

Comment:
Would be nice if these were configurable. Otherwise note in the docstring which external software is required.

Reply:
I'm opting to not allow too much flexibility, but I did add an explicit reference to the requirements in the docstring (both programs and also needed latex stuff).